network.zig

```diff
@@ -6,6 +6,7 @@ comptime {
 }
 
 const is_windows = std.builtin.os.tag == .windows;
+const is_mac = std.builtin.os.tag == .macosx;
```

Comment:
Maybe `std.builtin.os.isDarwin()` is better?  
I assume these fixes are needed for iOS, tvOS etc too

Reply:
You're absolutely right, thanks! Fixed it.